### PR TITLE
*: increase schema lease to make CI stable

### PR DESCRIPTION
**Pull request by tiancaiamao**
### What problem does this PR solve? 

Make CI more stable.

### What is changed and how it works?

Sometimes CI is not stable, especially in the race test.
It fails with some message like this

```
terror.go:357: [error] [domain:1]Information schema is out of date.
```

And the real reason is this:

```
[domain.go:361] ["loading schema takes a long time"] ["take time"=2.099572096s]
```

[log (1).txt](https://github.com/pingcap/tidb/files/3787450/log.1.txt)

### Check List 

Tests 

 - No code

Side effects

 - Possible performance regression
I test with  package and it's acceptable.
Before the change:
ok  	github.com/pingcap/tidb/ddl	19.372s
After the change:
ok  	github.com/pingcap/tidb/ddl	19.604s

### CODE REVIEW

**Comment by tiancaiamao** [comment]
PTAL @zimulala

File: ddl/db_test.go [hunk 1]
```diff
@@ -85,7 +85,6 @@ func setUpSuite(s *testDBSuite, c *C) {
 	var err error
 
 	s.lease = 100 * time.Millisecond
-	session.SetSchemaLease(s.lease)
```
Comment (zimulala) [inline ddl/db_test.go]: Many tests rely on the lease value, it is best to check the use of this value in the test.

Comment (tiancaiamao) [reply ddl/db_test.go]: This change affects db_test.go
I find the failed test is db_test.go, so I change it.
If there are other tests found to fail on SchemaLease, we can also update them.

**Comment by tiancaiamao** [comment]
The "Information schema out of data" we meet in the log is a result, not the root cause.
The root cause of this problem is still under investigating.

**Pull request closed without merging**